canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -40,9 +37,19 @@ $toolbar-divider-width: 1px;
 .toolbar-left-bar {
 	flex-shrink: 5000000; // Make sure left bar always shrinks first
 	display: block;
-	padding-right: 42px;
 	overflow: hidden;
 	align-items: center;
+	.toolbar-icon-label.before {
+		padding: $spacing-04;
+	}
+	.togglePalette-action,
+	.undo-action,.redo-action,
+	.cut-action,.copy-action,
+	.paste-action,
+	.createAutoComment-action,
+	.deleteSelectedObjects-action{
+		padding: 3px;
```

Comment:
You cant specify the styles like this for the specific toolbar items. These are some of the default ones that canvas provides, but applications can customize it with their own items. Any changes done to toolbar items need to be generic